<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckView</class>
 <widget class="QFrame" name="CheckView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1132</width>
    <height>72</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QRadioButton::indicator { 
	width: 48px;
	height: 48px;
}
/*单选框未选中样式*/
QRadioButton::indicator::unchecked {
	image: url(:/icons/uncheck.png);
}
/*单选框选中样式*/
QRadioButton::indicator::checked { 
	image: url(:/icons/check.png);
}
/*RadioButton和checkbox字体和间距设置*/
QRadioButton {
	spacing: 5px;
	font: 14pt &quot;黑体&quot;;
}

QSpinBox,QDoubleSpinBox {
	font:14pt &quot;黑体&quot;;
	min-height:50px;
	min-width:150px;
	border-radius:24;
	border:2px solid black;
	background-color:transparent;
}

/*spinbox 抬起样式*/
QTimeEdit::up-button,QDoubleSpinBox::up-button,QSpinBox::up-button {subcontrol-origin:border;
    subcontrol-position:right;
	image: url(:/icons/up_left.png);
    width: 50px;
    height: 50px;       
}
QTimeEdit::down-button,QDoubleSpinBox::down-button,QSpinBox::down-button {subcontrol-origin:border;
    subcontrol-position:left;
	image: url(:/icons/up_right.png);
    width: 50px;
    height: 50px;
}

/*按钮按下样式*/
QTimeEdit::up-button:pressed,QDoubleSpinBox::up-button:pressed,QSpinBox::up-button:pressed{subcontrol-origin:border;
    subcontrol-position:right;
    image: url(:/icons/up_left.png);
    width: 49px;
    height: 49px;       
}
  
QTimeEdit::down-button:pressed,QDoubleSpinBox::down-button:pressed,QSpinBox::down-button:pressed,QSpinBox::down-button:pressed{
    subcontrol-position:left;
    image: url(:/icons/up_right.png);
    width: 49px;
    height: 49px;   
}</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,0,0,0,0,0,1">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="closetBtn">
     <property name="toolTip">
      <string>Close/Connect</string>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
     <property name="icon">
      <iconset resource="../mainwindow.qrc">
       <normaloff>:/icons/disconnect.png</normaloff>:/icons/disconnect.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonMs">
     <property name="text">
      <string>Microscale</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinBoxChannel">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> Channel</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonCt">
     <property name="text">
      <string>Cuvette</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="checktBtn">
     <property name="toolTip">
      <string>Check</string>
     </property>
     <property name="text">
      <string>Check</string>
     </property>
     <property name="icon">
      <iconset resource="../mainwindow.qrc">
       <normaloff>:/icons/test.png</normaloff>:/icons/test.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="savetBtn">
     <property name="toolTip">
      <string>Save Image</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../mainwindow.qrc">
       <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="maintBtn">
     <property name="toolTip">
      <string>Main Borad</string>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="text">
      <string>Main Borad</string>
     </property>
     <property name="icon">
      <iconset resource="../mainwindow.qrc">
       <normaloff>:/icons/mainboard.png</normaloff>:/icons/mainboard.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="microtBtn">
     <property name="toolTip">
      <string>Check Microscale</string>
     </property>
     <property name="text">
      <string>Check Microscale</string>
     </property>
     <property name="icon">
      <iconset resource="../mainwindow.qrc">
       <normaloff>:/icons/macroscale.png</normaloff>:/icons/macroscale.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="cuvettetBtn">
     <property name="toolTip">
      <string>Check Cuvette</string>
     </property>
     <property name="text">
      <string>Check Cuvette</string>
     </property>
     <property name="icon">
      <iconset resource="../mainwindow.qrc">
       <normaloff>:/icons/cuvette.png</normaloff>:/icons/cuvette.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="temperaturetBtn">
     <property name="toolTip">
      <string>Temperature</string>
     </property>
     <property name="text">
      <string>Temperature</string>
     </property>
     <property name="icon">
      <iconset resource="../mainwindow.qrc">
       <normaloff>:/icons/temperature.png</normaloff>:/icons/temperature.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="messageLabel">
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
